client/state/plugins/installed/test/reducer.js

```diff
@@ -0,0 +1,239 @@
+/* eslint-disable max-len */
```

Comment:
Do you have a good reason for disabling `max-len` in this file? The long lines I see could easily be refactored onto separate lines.